CHANGELOG.md

```diff
@@ -2,7 +2,11 @@
 
 ## dev
 
-Version <dev> of the agent introduces improved error tracking functionality by associating a transaction id with each error, and uses more reliable network timeout logic.
+Version <dev> of the agent adds a new 'allow_all_headers' configuration parameter to permit the capturing of all HTTP headers, introduces improved error tracking functionality by associating a transaction id with each error, and uses more reliable network timeout logic.
+
+- **Feature: New allow_all_headers configuration parameter**
+
+  A new `allow_all_headers` configuration parameter brings parity with the Node.js agent and others. This configuration parameter defaults to a value of `false`. When set to `true`, and as long as the agent is not operating in high-security mode, all HTTP headers gleaned from a request will be captured and relayed to New Relic instead of the default core set of headers. All existing behavior for `.*attributes.include` and `.*attributes.exclude` configuration parameters will be respected for any desired filtration of the headers when `allow_all_headers` is enabled. This work was done in response to a feature request submitted by community member [@jamesarosen](https://github.com/jamesarosen). Thank you very much, @jamesarosen! [Issue#1029](https://github.com/newrelic/newrelic-ruby-agent/issues/1029)
```

Comment:
> brings parity with the Node.js agent and others.

Who are the others?

Reply:
@hannahramadan brought this up as well. `allow_all_headers` is in the specs, so I assumed that there were others. But after double checking it looks like it's just Ruby, Node.js, and Browser. I'm open to changing it.

Reply:
I'm in good company! I think removing "and others" would be helpful. It might also be helpful to link to documentation for the Node and/or the Browser agent's versions in this description.

Reply:
CHANGELOG entry updated in [16be793](https://github.com/newrelic/newrelic-ruby-agent/pull/2155/commits/16be7939b73a548c6f271657dd21c20743a5fcdc). Apparently I imagined the browser agent part, so I've removed "others" and just refer to Node. The best documentation I could find on the Node side for the feature was in its changelog.